scripts/jenkins/cloud/ansible/crowbar-update.yml

```diff
@@ -0,0 +1,95 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+---
+
+- import_playbook: _create-vcloud-inventory.yml
+
+- name: Update cloud - crowbar
+  hosts: "{{ cloud_env }}"
+  remote_user: root
+  gather_facts: True
+  vars:
+    task: "update"
+
+  pre_tasks:
+    - include_role:
+        name: rocketchat_notify
```

Comment:
Do we need this? I thought we agreed on removing the Rocked Notifications.

Reply:
@rsalevsky RC notifications are disabled by default, but we're following the same pattern used for other ansible roles, just in case anyone wants to use it explictily

Reply:
Okay works for me.